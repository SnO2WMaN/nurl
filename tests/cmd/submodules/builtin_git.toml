args = [
    "https://github.com/Koihik/LuaFormatter",
    "refs/tags/1.3.0",
    "--submodules",
    "--fetcher",
    "builtins.fetchGit",
]
