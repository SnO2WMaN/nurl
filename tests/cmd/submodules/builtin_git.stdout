builtins.fetchGit {
  url = "https://github.com/Koihik/LuaFormatter";
  ref = "refs/tags/1.3.0";
  submodules = true;
}